app/scripts/controllers/metamask-notifications/processors/process-onchain-notifications.test.ts

```diff
@@ -41,9 +41,10 @@ const rawNotificationTestSuite = rawNotifications.map(
 );
 
 describe('process-onchain-notifications - processOnChainNotification()', () => {
+  // @ts-expect-error This is missing from the Mocha type definitions
```

Comment:
Q - are our unit tests for controllers ran against mocha or jest? For example I thought these controllers are ran against jest unit tests here?
https://github.com/MetaMask/metamask-extension/blob/develop/jest.config.js#L53-L56

Reply:
Unfortunately our ESLint config is currently running both mocha and Jest lint rules on certain files. And on certain test files, it's running neither. This is one of the ESLint configuration issues that this PR helps to unblock. It was difficult to fix the scope issues with how the ESLint rules were applied without also surfacing these type errors.

Reply:
Yep this is fair. Lets get this type coverage in first :)